classquiz/config.py

```diff
@@ -32,23 +33,25 @@ class Settings(BaseSettings):
     redis: RedisDsn = "redis://localhost:6379/0?decode_responses=True"
     skip_email_verification: bool = False
     db_url: str | PostgresDsn = "postgresql://postgres:mysecretpassword@localhost:5432/classquiz"
-    hcaptcha_key: str | None = None
+    hcaptcha_key: str = "3852831a-48b4-4236-b0de-9769a1998468"
     recaptcha_key: str | None = None
-    mail_address: str
-    mail_password: str
-    mail_username: str
-    mail_server: str
-    mail_port: int
-    secret_key: str
+    #    mail_address: str
+    mail_address: str = "778@madeupaddress.com"
+    #    mail_password: str
+    mail_password: str = "mail_passwordstring"
+    mail_username: str = "mail_userstring"
+    mail_server: str = "mail_serverstring"
+    mail_port: int = 0
+    secret_key: str = "secret_keystring"
     access_token_expire_minutes: int = 30
     cache_expiry: int = 86400
-    sentry_dsn: str | None
+    sentry_dsn: str = ""
     meilisearch_url: str = "http://127.0.0.1:7700"
     meilisearch_index: str = "classquiz"
-    google_client_id: Optional[str]
-    google_client_secret: Optional[str]
-    github_client_id: Optional[str]
-    github_client_secret: Optional[str]
+    google_client_id: str = "google_client_idstring"
+    google_client_secret: str = "google_client_secretstring"
+    github_client_id: str = "github_client_idstring"
+    github_client_secret: str = "github_client_secretstring"
```

Comment:
Pls don't set a non-working default here

Reply:
They may be non-working, but...if I leave it as originally was....I get:

google_client_id
  Field required [type=missing, input_value={'storage_backend': 'loca...e_path': '/tmp/storage'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.7/v/missing
google_client_secret
  Field required [type=missing, input_value={'storage_backend': 'loca...e_path': '/tmp/storage'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.7/v/missing
github_client_id
  Field required [type=missing, input_value={'storage_backend': 'loca...e_path': '/tmp/storage'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.7/v/missing
github_client_secret
  Field required [type=missing, input_value={'storage_backend': 'loca...e_path': '/tmp/storage'}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.7/v/missing

Reply:
https://docs.pydantic.dev/2.7/migration/#required-optional-and-nullable-fields